packages/recorder/src/index.tsx

```diff
@@ -17,11 +17,11 @@
 import '@web/common.css';
 import { applyTheme } from '@web/theme';
 import '@web/third_party/vscode/codicon.css';
-import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { Main } from './main';
 
 (async () => {
   applyTheme();
+  // we'd like to migrate this to React 18, but concurrent mode seems to break some of our tests.
```

Comment:
```suggestion
  // TODO: we'd like to migrate this to React 18, but concurrent mode seems to break some of our tests.
```

then its easier to grep in the future.